pkg/mobile/serviceInstances.go

```diff
@@ -17,5 +18,37 @@ func NewServiceInstanceLister(scClient scv1beta1.ServicecatalogV1beta1Interface)
 
 func (lister *ServiceInstanceListerImpl) List(namespace string) (*ServiceInstanceList, error) {
 	listOpts := v1.ListOptions{}
-	return lister.scClient.ServiceInstances(namespace).List(listOpts)
+	getOpts := v1.GetOptions{}
+	serviceInstanceList := ServiceInstanceList{}
+
+	serviceInstances, err := lister.scClient.ServiceInstances(namespace).List(listOpts)
+	if err != nil {
+		return nil, err
+	}
+
+	for _, si := range serviceInstances.Items {
+		sc, err := lister.scClient.ClusterServiceClasses().Get(si.Spec.ClusterServiceClassRef.Name, getOpts)
```

Comment:
am I right to say that the `mobile-service` tag is only available on the `ServiceClass`, but not on the `ServiceInstance`?

Reply:
Yeah it is only available on the `ServiceClass`.
There was a problem with tagging/labeling the `ServiceInstance` on creation

Reply:
Right, ok. In this case you will need to update the test to add an instance of `ServiceClass`. The test doesn't have one ATM. That's why the build is failing.

Reply:
:+1: I'll update the test